docs/paper/admin/reference/configuration/global-configuration.mdx

```diff
@@ -0,0 +1,415 @@
+      to mitigate this desync by sending precise locations for entities involved in
+      collisions. Having this enabled will use more bandwidth; however, in the majority
+      of cases, this is a worthy tradeoff'
+commands:
+  fix-target-selector-tag-completion:
+    default: 'true'
+    description: 'Workaround for a client bug preventing entity type tag suggestions
+      from functioning in target selectors. Resolves [MC-235045](https://bugs.mojang.com/browse/MC-235045)'
+  suggest-player-names-when-null-tab-completions:
+    default: 'true'
+    description: 'Instructs the server to return a list of players when tab-completing
+      when there are no other completions available'
+  time-command-affects-all-worlds:
+    default: 'false'
+    description: 'Whether the /time command should act on all worlds or just the sender''s
+      current world'
+console:
+  enable-brigadier-completions:
+    default: 'true'
+    description: 'Enables Mojang''s Brigadier (advanced) command completions in the
+      server console'
+  enable-brigadier-highlighting:
+    default: 'true'
+    description: 'Enables Mojang''s Brigadier highlighting in the server console'
+  has-all-permissions:
+    default: 'false'
+    description: 'Whether the console command sender has all permissions'
+item-validation:
+  display-name:
+    default: '8192'
+    description: 'The maximum length of an item''s display name in characters'
+  lore-line:
+    default: '8192'
+    description: 'The maximum length of a lore line in characters'
+  resolve-selectors-in-books:
+    default: 'false'
+    description: 'Whether to resolve selectors in books. With this enabled,
+      players given creative mode will be able to crash the server in yet another way'
+  book:
+    author:
+      default: '8192'
+      description: 'The maximum length of a book''s author in characters'
+    page:
+      default: '16384'
+      description: 'The maximum length of a book''s page in characters'
+    title:
+      default: '8192'
+      description: 'The maximum length of a book''s title in characters'
+  book-size:
+    page-max:
+      default: '2560'
+      description: 'The max number of bytes a single page in a book can contribute
+        to the allowed byte total for a book'
+    total-multiplier:
+      default: '0.98'
+      description: 'Each page has this multiple of bytes from the last page as its
+        contribution to the allowed byte total for a book (with the first page being
+        having a multiplier of 1.0)'
+logging:
+  deobfuscate-stacktraces:
+    default: 'true'
+    description: 'Whether to remap Spigot mapped stacktraces to Mojang mappings in
+      logging. Has no impact on Mojang mapped servers'
+  log-player-ip-addresses:
+    default: 'true'
+    description: 'Whether player IP addresses should be logged by the server. This
+      does not impact the ability of plugins to log the IP addresses of players'
+  use-rgb-for-named-text-colors:
```

Comment:
This section doesn't exists anymore: https://github.com/PaperMC/Paper/blob/master/patches/server/0005-Paper-config-files.patch#L1474 since the new ansi encoder